game/src/main/scala/io/aigar/game/Cell.scala

```diff
@@ -104,7 +109,7 @@ class Cell(val id: Int, player: Player, startPosition: Vector2 = new Vector2(0f,
 
   def state: serializable.Cell = {
     serializable.Cell(id,
-                      round(mass).toInt,
```

Comment:
Then the other one shouldn't be a `Double`. Can we fix it, while we're modifying this?